src/loadReferanceFa.jl

```diff
@@ -0,0 +1,10 @@
+#a function to load in referance data
+function loadReference(reference)
+    f = open(string("test/sample_data/",  reference))
```

Comment:
this is not geenralized, it can only load reference, this should take the full path as input